.github/workflows/docker-build.yml

```diff
@@ -15,9 +15,10 @@ jobs:
         id: set-tag
         run: |
           SHORT_SHA=$(git rev-parse --short HEAD)
-          if [[ "${{ github.ref }}" == "refs/heads/main" ]]; then
+          TAG_VERSION=$(echo ${{ github.ref }} | sed 's,refs/tags/v,,g')
```

Comment:
`sed` is fine and the cool kids use it. But you know what's cooler? Using what the github API gives us. Please use github.ref_name. I also recommend taking a peek into github documentation:

- https://docs.github.com/en/actions/learn-github-actions/contexts#github-context
- https://docs.github.com/en/actions/learn-github-actions/environment-variables#default-environment-variables

Reply:
Please take another good look. If github.ref_name would do string manipulations recently, I would actually have to look at the docs again

I'm not using `sed` for fun, but to strip the prefix "v" from the tag on refs/tags

Reply:
Then make it even simpler. Create a separate workflow when a push with a tag occurs. 

If you're concerned about a bad actor pushing a branch name that matches semver, there should be a way to block this in github. But first would need to be a member of this repo, I guess.

Sed is used for live replacement of strings / files in Linux. Seeing it here just to grab the version from a ref name is an intuitive way to use it, but my concern is bigger. I ask you to reconcider how you approach the whole "release of a stable artifact". Use the primitives that github offers (like github.ref_type to separate between tags/branches) and eventually even use prepared github actions.

Reply:
- https://github.com/caas-team/py-kube-downscaler/pull/21/commits/0a8c46e07f6f9d722c685efad1e670767c3fb908